controllers/rcj_soccer_ball/rcj_soccer_ball.py

```diff
@@ -8,5 +8,5 @@
 data = [True]  # Packet cannot be empty
 packet = struct.pack("?", *data)
 
-while robot.step(32) != -1:
+while robot.step(8) != -1:
```

Comment:
Why is the time step reduced to 8? Did 32 caused any problems?

Reply:
Hello @Adman,

Yes, since the physics computation is now a lot more complicated than before, changing the value for the basic time step of the world would drastically affect the way that robots move and perform. After testing, we found that setting the basic time step of the world to 8 is the most ideal. When we set the basic time step to 32, the physics-related warning message `WARNING: The current physics step could not be computed correctly. Your world may be too complex. If this problem persists, try simplifying your bounding object(s), reducing the number of joints, or reducing WorldInfo.basicTimeStep.` appeared more frequently. On the other hand, when we set the basic time step to a value lower than 8, the robots no longer moved as we expected.

Let us know if you have any other questions regarding this issue!

Team Reset